dialog.cpp

```diff
@@ -74,7 +74,7 @@ Dialog::Dialog(QWidget *parent) :
     setWindowTitle(QSL("LXQt Runner"));
     setAttribute(Qt::WA_TranslucentBackground);
 
-    connect(LXQt::Settings::globalSettings(), &LXQt::GlobalSettings::iconThemeChanged, this, &Dialog::reload);
+    connect(LXQt::Settings::globalSettings(), &LXQt::GlobalSettings::iconThemeChanged, this, QOverload<>::of(&Dialog::update));
```

Comment:
Just one note... we require c++14, so you can use `qOverload<>(...` here.

Reply:
Ok